.github/workflows/release-binaries.yml

```diff
@@ -130,9 +129,6 @@ jobs:
           echo ccache=sccache >> $GITHUB_OUTPUT
         fi
 
-        # Detect necessary CMake flags
-        echo "enable-pgo=false" >> $GITHUB_OUTPUT
-        target_cmake_flags="-DLLVM_RELEASE_ENABLE_PGO=OFF"
```

Comment:
I'm assuming dropping this flag is what actually is what enables the release builds to be done with PGO?

I'm not seeing how this plumbing works though. This flag doesn't seem to exist anywhere else in the codebase (although Github search could just be lying to me).

Reply:
The default setting is here: https://github.com/llvm/llvm-project/blob/main/clang/cmake/caches/Release.cmake#L37

Reply:
Ah, it's defined purely in the cache file. Seems good to me then. Thanks for the explanation!